src/discussions/posts/post/PostLink.jsx

```diff
@@ -76,8 +76,10 @@ function PostLink({
               />
             </div>
           </div>
-          {/* eslint-disable-next-line react/no-danger */}
-          <div dangerouslySetInnerHTML={{ __html: post.previewBody }} />
+          <div className="text-break">
+            {/* eslint-disable-next-line react/no-danger */}
+            <div dangerouslySetInnerHTML={{ __html: post.previewBody }} />
+          </div>
```

Comment:
I'm not sure this is needed. This text will not overflow since it's limited in length to 35 chars. 

That said, if we'd still like to be safe, this approach will still cause multiple lines to be displayed which is not what we want. The following would work better:

```suggestion
          {/* eslint-disable-next-line react/no-danger */}
          <div className="text-truncate" dangerouslySetInnerHTML={{ __html: post.previewBody }} />
```

Reply:
Updated